.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Splendid! Proper directories were ignored 🙌

Reply:
I will add a docs folder to put in there the build html and bundle.js as well. Since GH pages ignores dist folder. I then will add dist to this file.